rust/test/ca/inc/commands_aggregator/basic.rs

```diff
@@ -128,6 +128,35 @@ tests_impls!
 
     a_id!( Ok( () ), ca.perform( "-command" ) );
   }
+
+  fn dot_command()
+  {
+    let ca = CommandsAggregator::former()
+    .grammar(
+    [
+      wca::Command::former()
+      .hint( "hint" )
+      .long_hint( "long_hint" )
+      .phrase( "cmd.first" )
+      .form(),
+      wca::Command::former()
+      .hint( "hint" )
+      .long_hint( "long_hint" )
+      .phrase( "cmd.second" )
+      .form(),
+    ])
+    .executor(
+    [
+      ( "cmd.first".to_owned(), Routine::new( | _ | { println!( "Command" ); Ok( () ) } ) ),
+      ( "cmd.second".to_owned(), Routine::new( | _ | { println!( "Command2" ); Ok( () ) } ) ),
+    ])
+    .build();
+
+    a_id!( Ok( () ), ca.perform( "." ) );
+    a_id!( Ok( () ), ca.perform( ".cmd." ) );
+
+    a_id!( Ok( () ), ca.perform( ".c." ) );
```

Comment:
I think it should be an error. It is because the command doesn't exists in the dictionary. The command `.` is special case that should show help and throw no error.